ci/test_wheel.sh

```diff
@@ -41,6 +37,6 @@ else
        -v \
        --import-mode=append \
        --benchmark-disable \
-       -k "not test_property_graph_mg and not test_bulk_sampler_io" \
-       ./python/${package_name}/${python_package_name}/tests
+       -k "test_pagerank_non_convergence" \
+       ./python/${package_name}/${python_package_name}/tests/link_analysis/
```

Comment:
Why was this changed?

Also, we can probably delete `NX_CUGRAPH_USE_COMPAT_GRAPHS=False` a few lines down.

Reply:
Ah, sorry. This was not a change intended for this PR